### ISSUE TITLE
Error on starting with 0.2.3

### ISSUE THREAD

**Issue by rchl** [daveleroy/sublime_debugger#61]
I'm on ST4 dev version (haven't tested ST3).
```
removed old modules: ['Debugger.__init__']
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/Lib/python33/sublime_plugin.py", line 274, in reload_plugin
    m = importlib.import_module(modulename)
  File "./python3.3/importlib/__init__.py", line 90, in import_module
  File "<frozen importlib._bootstrap>", line 1584, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1565, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1532, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 584, in _check_name_wrapper
  File "<frozen importlib._bootstrap>", line 1022, in load_module
  File "<frozen importlib._bootstrap>", line 1003, in load_module
  File "<frozen importlib._bootstrap>", line 560, in module_for_loader_wrapper
  File "<frozen importlib._bootstrap>", line 868, in _load_module
  File "<frozen importlib._bootstrap>", line 313, in _call_with_frames_removed
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/start.py", line 13, in <module>
    from .modules.debugger.commands import DebuggerCommand
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/modules/debugger/commands.py", line 3, in <module>
    from .debugger import Debugger
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/modules/debugger/debugger.py", line 20, in <module>
    from .debugger_session import (
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/modules/debugger/debugger_session.py", line 8, in <module>
    from .terminals import (
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/modules/debugger/terminals/__init__.py", line 18, in <module>
    from .terminal_build  import (
  File "/Users/me/Library/Application Support/Sublime Text/Packages/Debugger/modules/debugger/terminals/terminal_build.py", line 55, in <module>
    class DebuggerBuildExecCommand(Default.exec.ExecCommand):
AttributeError: 'module' object has no attribute 'exec'
```

**Comment by rchl** [comment]
It looks like that's because `Default` package is now running in py38 runtime so it's not available in 33 runtime anymore.

**Comment by daveleroy** [comment]
Should be fixed in 0.2.4

**Comment by rchl** [comment]
WFM now. Thank you.
